virttest/qemu_devices/qdevices.py

```diff
@@ -417,8 +433,8 @@ def __init__(self, dev_type, params=None, aobject=None,
         else:
             self.__backend = None
 
-    def cmdline(self):
-        """ :return: cmdline command to define this device """
+    def _cmdline_raw(self):
```

Comment:
it seems we don't have a _cmdline_json implementation for class QObject nor its parent QCustomDevice.